bubble-wrap-style.yaml

```diff
@@ -2631,6 +2440,34 @@ layers:
                 #             # you need to match any custom shield to the vector tile `network` values
                 #             #priority: 48
                 #             visible: global.sdk_road_shields
+                US-AL:
+                    # Match Alabama state highways: `US:AL`
+                    filter: |
```

Comment:
Would be better to state general USA matching list (like we do with international so we don't need to restate the draw texture sprite bit repetitively... and then only the states with custom text offsets should get a sublayer.